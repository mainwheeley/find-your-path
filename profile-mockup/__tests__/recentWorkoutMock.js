import React from 'react';
import { View, Text, StyleSheet } from 'react-native';
import RecentWorkouts from '../comps/components/recentWorkouts';
import { shallow } from 'enzyme';
describe('Profile Bar', function() {
    it('renders correctly', () => {
	const tree = shallow(<RecentWorkouts />)
	expect(tree).toMatchSnapshot();
    });
    it('length 1', function() {
	const tree = shallow(<RecentWorkouts />)
	expect(tree.length).toBe(1);
    });
});
