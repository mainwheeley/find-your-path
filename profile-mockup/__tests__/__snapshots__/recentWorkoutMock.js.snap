// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Bar renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <View
    style={
      Object {
        "flex": 1,
        "justifyContent": "center",
        "margin": 10,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "borderBottomColor": "#000",
            "fontSize": 20,
            "fontWeight": "bold",
            "height": 44,
            "padding": 10,
          }
        }
      >
        Most recent workout: 
      </Text>
    </View>
    <FlatList
      ItemSeparatorComponent={[Function]}
      data={
        Array [
          Object {
            "key": "1",
            "startp": "134 W State St.",
            "totmiles": 6,
            "type": "Bike",
          },
        ]
      }
      disableVirtualization={false}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      numColumns={1}
      onEndReachedThreshold={2}
      renderItem={[Function]}
      scrollEventThrottle={50}
      updateCellsBatchingPeriod={50}
      windowSize={21}
    />
  </View>,
  "nodes": Array [
    <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "center",
          "margin": 10,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fff",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Object {
              "borderBottomColor": "#000",
              "fontSize": 20,
              "fontWeight": "bold",
              "height": 44,
              "padding": 10,
            }
          }
        >
          Most recent workout: 
        </Text>
      </View>
      <FlatList
        ItemSeparatorComponent={[Function]}
        data={
          Array [
            Object {
              "key": "1",
              "startp": "134 W State St.",
              "totmiles": 6,
              "type": "Bike",
            },
          ]
        }
        disableVirtualization={false}
        horizontal={false}
        initialNumToRender={10}
        keyExtractor={[Function]}
        maxToRenderPerBatch={10}
        numColumns={1}
        onEndReachedThreshold={2}
        renderItem={[Function]}
        scrollEventThrottle={50}
        updateCellsBatchingPeriod={50}
        windowSize={21}
      />
    </View>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <RecentWorkouts />,
    "_instance": RecentWorkouts {
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": Object {},
      "updater": Updater {
        "_renderer": [Circular],
      },
      "workoutSeparator": [Function],
    },
    "_newState": null,
    "_rendered": <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "center",
          "margin": 10,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fff",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Object {
              "borderBottomColor": "#000",
              "fontSize": 20,
              "fontWeight": "bold",
              "height": 44,
              "padding": 10,
            }
          }
        >
          Most recent workout: 
        </Text>
      </View>
      <FlatList
        ItemSeparatorComponent={[Function]}
        data={
          Array [
            Object {
              "key": "1",
              "startp": "134 W State St.",
              "totmiles": 6,
              "type": "Bike",
            },
          ]
        }
        disableVirtualization={false}
        horizontal={false}
        initialNumToRender={10}
        keyExtractor={[Function]}
        maxToRenderPerBatch={10}
        numColumns={1}
        onEndReachedThreshold={2}
        renderItem={[Function]}
        scrollEventThrottle={50}
        updateCellsBatchingPeriod={50}
        windowSize={21}
      />
    </View>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <RecentWorkouts />,
}
`;
